src/main/java/http/response/Response.java

```diff
@@ -46,4 +46,17 @@ public void setBody(byte[] body) {
     }
 
 
+
+    public void sendResponse () throws IOException {
+         if (this.body == null){  this.body = "".getBytes(); };
+        String formattedRequest =
```

Comment:
this should be `formattedResponse`?

Reply:
It will change once I create a build response class I just wrote this to get it to work as is now.